README.md

```diff
@@ -27,34 +34,67 @@ De esta forma, el total de tiempo empleado sería 34 mins, mientras que de cualq
 ### Perfil de los usuarios
 
 * [Perfil usuarios](docs/perfil_usuarios.md)
+
```

Comment:
Meter estos espacios para justificar un commit sólo hace que yo tenga más trabajo al revisar. Por favor, no lo hagáis.

Reply:
De acuerdo